src/blog-utils/index.js

```diff
@@ -1,17 +1,22 @@
 /* @flow */
+/* global fetch */
 import * as React from 'react';
 import { renderToStaticMarkup } from 'react-dom/server';
 import { Provider } from 'react-redux';
 import { ConnectedRouter } from 'connected-react-router';
 import { createMemoryHistory } from 'history';
+import 'isomorphic-fetch';
 
 import I18nProvider from 'amo/i18n/Provider';
 import { makeI18n } from 'amo/i18n/utils';
 import { setClientApp, setLang } from 'amo/reducers/api';
 import createStore from 'amo/store';
 import Footer from 'amo/components/Footer';
+import { createInternalAddon } from 'amo/reducers/addons';
 
-import './styles.scss';
+import StaticAddonCard from './StaticAddonCard';
+
+const AMO_BASE_URL = 'https://addons.mozilla.org';
```

Comment:
should this be in config so it can be overridden to point to a local/dev/stage url?

Reply:
I'd like to try to NOT bundle the frontend config in the lib if possible, which is why it's the prod url only. Also, I don't know exactly how deployments will work in addons-blog, so this might change..